### ISSUE TITLE
Possibly misleading license info

### ISSUE THREAD

**Issue by EricRa** [yl4579/StyleTTS2#37]
This repo claims to use an MIT license, but there are additional license requirements buried in the readme file:

> Before using these models, you agree to inform the listeners that the speech samples are synthesized by StyleTTS 2 models, unless you have the permission to use the voice you synthesize. That is, when using it for voice cloning, you also agree to only use voices whose speakers grant the permission to have their voice cloned, either directly or by license before making synthesized voices pubilc.

This is very misleading because someone simply checking the license file before using the repo would make the assumption that only the MIT license requirements apply.  

I understand that the intention is probably to license the code in the repo as MIT, while having additional license requirements for the pre-trained models.  However, because the only apparent way to get the models is a Google Drive link contained in the repo, it still seems misleading.  Additionally, the wording above suggests that this license requirement applies not only to the pre-trained models but also to any StyleTTS 2 models created using the code in the repo.

IMO, any additional license requirements for using the code in the repo as intended should be mentioned in the repo license file.

If I am misunderstanding something, I take no offense to you simply closing this issue without comment.

**Comment by yl4579** [comment]
These rules are to address the ethics reviews we received in NeurIPS 2023. The reviewer expressed concerns over the possibility of deception:
![image](https://github.com/yl4579/StyleTTS2/assets/71044569/06c599a8-54c0-40dd-87a7-aa5ea4c23970)
and we have addressed these issues by asking the users to abide by certain rules to use the pre-trained models, which the reviewer accepts:
![image](https://github.com/yl4579/StyleTTS2/assets/71044569/0edc5a1c-7b7f-4061-a8fa-991c565b177b)
Clearly the source of the concern is the model, not the code itself. For example, pre-trained model on LJSpeech cannot be used for deception, but models trained on larger datasets with tons of speakers can be used for deception. This is why we set the ethical rules for the models, not the code. The codebase itself is MIT, and any model you trained yourself does not have to follow these rules. 

However, I understand this can be misleading, and I appreciate your concern. Given the ethics review we receive and our intention to address them on the rebuttal, what are some better ways you think we can do to make it more consistent and clear?

**Comment by cmp-nct** [comment]
I do not see this as a MIT violation in any way, the code is MIT without any extras..

As far as I understood the models are free with that one exception and you can retrain your own ones with the included dataset that's public domain.

**Comment by gmorenz** [comment]
It might help to clarify "synthesized by StyleTTS 2 models" as "synthesized by the pretrained StyleTTS 2 models". It seems that the entire project is titled StyleTTS2, so "StyleTTS 2 models" could be read as meaning all models trained using the source code, whether or not they are trained from scratch. I don't think that's what you intended or the most reasonable reading, but I think it's where the confusion comes from.

Separately I believe it would be a good idea to explicitly grant rights to use the pretrained models subject to the terms you feel are necessary. Similar to how the MIT explicitly grants rights to use the source code. The default on copyright is "all rights reserved", so without a grant of rights using these models seems legally dubious. That's also the means by that the restrictions would typically be legally binding, a licence that says "you can <do things> provided you comply with <restrictions>". For example see the MIT license, where the restriction is simply including the notice in all copies or substantial portions of the software.

I'm not a lawyer though, and I don't think I can ethically propose precise terms - usually writing bespoke licenses like this is something done by lawyers.

**Comment by EricRa** [comment]
Agreed with gmorenz on both points.  Regarding the first point, it is not clear to me from the wording whether the additional "permission" requirement applies only to the pre-trained models.  It reads to me as though it applies to all models trained using the source code in the repo.  

If the intention is that this only applies to the pre-trained models, I don't understand the relevance of this statement

> That is, you agree to only use voices whose speakers grant the permission to have their voice cloned, either directly or by license before making synthesized voices pubilc

Presumably, you already have permission to use the voices of the people in your pre-trained models.  

Moving the models to a separate repo with a separate license file might help.  I realize you can't host the models on Github, but they could be linked from the model repo.  This way, you could specify license terms for the pre-trained models separate from the code.  I do not know if this is desirable or the best solution, just a possible suggestion.

My intention is not to be antagonistic, so this all doesn't come across that way.  The project looks fantastic.  I just think devs both in commercial and hobbyist settings would find it very useful to have a clear set of terms on the license requirements involved when using the code/models in their projects.

**Comment by yl4579** [comment]
@gmorenz Thanks for your suggestions. I have changed the wordings in README per your suggestion. I meant to be only the pre-trained models. We received this ethnics review after we stated that we would open source the code and pre-trained models, and apparently the LibriTTS model capable of voice cloning triggered the ethnics review flags, so our intention is mainly toward the pre-trained model weights, not the code itself. 

@EricRa I think the confusion comes from the voice cloning part. LibriTTS is a dataset that follows CC BY 4.0, so anyone has the right to use the voices inside the training set. However, the model can also synthesize in voices not seen during training, which is why I have to impose this rule to address some ethical issues such as deception. If you do not use the pre-trained models, however, you do not have to abide by these rules.
